### PR TITLE
define etag as a reserved field

### PR DESCRIPTION
**Pull request by zeitlinger**

### CODE REVIEW

File: chapters/common-data-types.adoc [hunk 1]
```diff
@@ -85,6 +85,8 @@ type `Node`)
 * `type`: the kind of thing this object is. If used, the type of this
 field should be a string. Types allow runtime information on the entity
 provided that otherwise requires examining the Open API file.
+* `etag`: the etag of an <<158, embedded sub-resource>>.
+It may be used to carry the etag for subsequent PUT calls (see <<etag-in-result-entities>>).
```
Comment (tkrop) [inline chapters/common-data-types.adoc]: I'm not absolutely sure, but I would expect this to mention the `ETag`-header explicitly (may be also linked to the RFC) and refer to the usage pattern defined in https://opensource.zalando.com/restful-api-guidelines/#182.

Can you see that the latter #182 links correctly to <<etag-in-result-entities>>?

File: chapters/best-practices.adoc [hunk 2]
```diff
@@ -55,6 +55,7 @@ or if there was an update since the GET and the entity's etag has changed:
 ==== Cons
 * Many additional requests are necessary to build a meaningful front-end
 
+[[etag-in-result-entities]]
```
Comment (tkrop) [inline chapters/best-practices.adoc]: :+1:

File: chapters/common-data-types.adoc [hunk 3]
```diff
@@ -85,6 +85,8 @@ type `Node`)
 * `type`: the kind of thing this object is. If used, the type of this
 field should be a string. Types allow runtime information on the entity
 provided that otherwise requires examining the Open API file.
+* `etag`: the etag of an <<158, embedded sub-resource>>.
```
Comment (ddufourboivin-zalando) [inline chapters/common-data-types.adoc]: Should it be writted _ETag_ instead of _etag_?

File: chapters/common-data-types.adoc (hunk 1 above)
Comment (zeitlinger) [reply chapters/common-data-types.adoc]: @tkrop good idea - I've linked 182 (which in return links to the RFC), made sure the links work, and corrected the casing of ETag throughout the guidelines.

**Comment by whiskeysierra** [comment]
:+1:

**Comment by ddufourboivin-zalando** [comment]
👍

**Comment by tkrop** [comment]
:+1:

**Pull request merged**